modules/post-install-edge-aws-extend-machineset.adoc

```diff
@@ -50,3 +50,16 @@ $ aws ec2 describe-availability-zones --region <value_of_Region> \// <1>
     }
 ]
 ----
++ 
+.Example output for Wavelength Zone `us-east-1-wl1`
+[source,terminal]
+----
+[
+    {
+        "ZoneName": "us-east-1-wl1",
```

Comment:
Hi @mtulio . Are these Wavelength Zone values correct?

I referenced https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/describe_availability_zones.html

Reply:
No, this is a zone group name, or `group-name` in the linked doc.

Reply:
The example for Wavelength Zone in the AWS documentation is incorrect, both in the linked by you (boto3) and in the official product documentation:  https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeAvailabilityZones.html#API_DescribeAvailabilityZones_RequestParameters

I just opened an issue to AWS folks fix it, and submitted the feedback in AWS docs as it does not provide a channel to track issues.
https://github.com/boto/boto3/issues/3994

Reply:
Thanks, @mtulio . I've updated WZ example. Thanks for identifing the error and for raising a ticket. Great catch!